src/lib.rs

```diff
@@ -317,6 +317,8 @@ impl Default for BlendMode {
 pub struct DrawConfig {
     /// The scale of the drawn texture in drawn pixels per source pixel.
     pub scale: (u32, u32),
+    /// The rotation of the drawn texture in degrees.
+    pub rotation: i32,
```

Comment:
```suggestion
    ///
    /// Rotating by anything more precise than 90 steps can cause the source image
    /// to be slightly distorted, especially if it has a very low resolution.
    pub rotation: i32,
```
at least that's what I expect